check_style.xml

```diff
@@ -134,6 +134,8 @@
         <module name="ModifierOrder"/>
         <module name="EmptyLineSeparator">
             <property name="allowNoEmptyLineBetweenFields" value="true"/>
+            <property name="allowMultipleEmptyLines" value="false"/>
```

Comment:
Why change this?

Reply:
Some bad styles are founded by yanzh but not by checkstyle, so I enhanced the rules